### PR TITLE
refactor(ethereum-common): Add documentation comments

### PR DESCRIPTION
**Pull request by gshep**
@gear-tech/dev

### CODE REVIEW

File: ethereum-common/src/beacon/light.rs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Module contains lightened versions of the entities.
+//! Module contains lightened versions of the entities.
```
Comment (mertwole) [inline ethereum-common/src/beacon/light.rs:1]: Please clarify here what means `lightened` I suppose it means that some merkle sub-trees are precomputed?

File: ethereum-common/src/utils.rs [hunk 2]
```diff
@@ -34,43 +38,51 @@ impl fmt::Display for MerkleProofError {
 
 impl core::error::Error for MerkleProofError {}
 
+/// Contains Merkle proof for the provided receipt.
 pub struct MerkleProof {
     pub proof: Vec<Vec<u8>>,
```
Comment (mertwole) [inline ethereum-common/src/utils.rs:43]: Are there some specs on the format of this merkle proof in the Ethereum docs?

**Review by mertwole — APPROVED** [review]

File: ethereum-common/src/beacon/light.rs (hunk 1 above)
Comment (gshep) [reply ethereum-common/src/beacon/light.rs:1]: Good catch, thanks! Will add the note

File: ethereum-common/src/utils.rs (hunk 2 above)
Comment (gshep) [reply ethereum-common/src/utils.rs:43]: Frankly speaking I didn't search for that since we're using `trie-db` crate to generate/verify Merkle-proofs.

**Pull request merged**